hbase-plugins/src/main/java/co/cask/hydrator/plugin/sink/HBaseSink.java

```diff
@@ -69,7 +66,8 @@
 @Description("HBase Batch Sink")
 public class HBaseSink extends ReferenceBatchSink<StructuredRecord, NullWritable, Mutation> {
 
-  private static final Logger LOG = LoggerFactory.getLogger(HBaseSink.class);
+  private static final Logger logger = LoggerFactory.getLogger(HBaseSink.class);
```

Comment:
why changed the LOG variable to logger? LOG is a consistent variable name used across the CDAP code.

Reply:
LOG was giving checkstyle error.